docs/install.rst

```diff
@@ -90,9 +90,14 @@ match your environment: ::
 
     ./securedrop-admin sdconfig
 
-The script will automatically validate the answers you provided, and display
-error messages if any problems were detected. The answers you provided will be
-written to the file ``install_files/ansible-base/group_vars/all/site-specific``.
+An example of one such prompt would be to set the ``daily reboot time``.
```

Comment:
It does not seem to be useful to duplicate the example.

Reply:
You mean duplicates between admin and install documentation?

Reply:
Yes